src/situations/accueil/vues/accueil.vue

```diff
@@ -4,65 +4,167 @@
+        <div
+          :class="{ desactivee: precedentDesactivee}"
+          class="bouton-et-etiquette"
+        >
+          <button
+            :disabled="precedentDesactivee"
+            class="bouton-mission"
+            @click="indexBatiment--"
+           >
+            <img :src="precedent">
+          </button>
+          <span>{{ traduction('accueil.precedent') }}</span>
+        </div>
+        <a
+          v-if="situationActuelle"
+          :href="situationActuelle.chemin"
+          class="bouton-arrondi"
+        >
+          {{ traduction('accueil.commencer', { situation: situationActuelle.nom }) }}
+        </a>
+        <div
+          :class="{ desactivee: suivantDesactivee}"
+          class="bouton-et-etiquette gauche"
+        >
+          <button
+            :disabled="suivantDesactivee"
+            class="bouton-mission"
+            @click="indexBatiment++"
+           >
+            <img :src="suivant">
+          </button>
+          <span>{{ traduction('accueil.suivant') }}</span>
+        </div>
+      </div>
       <formulaire-identification :force-campagne="forceCampagne" />
     </div>
   </div>
 </template>
 
 <script>
-import { mapState } from 'vuex';
+import { mapState, mapGetters } from 'vuex';
 import 'accueil/styles/accueil.scss';
+import 'commun/styles/cadre.scss';
+import 'commun/styles/actions.scss';
+import 'commun/styles/bouton.scss';
 import FormulaireIdentification from './formulaire_identification';
 import AccesSituation from 'accueil/vues/acces_situation';
-import Progression from 'accueil/vues/progression';
 import BoiteUtilisateur from 'commun/vues/boite_utilisateur';
 
+const LARGEUR_SCENE = 1008;
+const LARGEUR_BATIMENT = 411;
+const DECALAGE_INITIAL = LARGEUR_SCENE / 2 - LARGEUR_BATIMENT / 2;
+const ESPACEMENT_BATIMENT = (LARGEUR_SCENE - 1.5 * LARGEUR_BATIMENT) / 2;
+
+export const CLE_NIVEAU_PRECEDENT = 'niveauPrecedent';
+
 export default {
-  components: { FormulaireIdentification, AccesSituation, Progression, BoiteUtilisateur },
+  components: { FormulaireIdentification, AccesSituation, BoiteUtilisateur },
 
   data() {
     const parsedUrl = new URL(window.location.href);
     return {
       fondAccueil: `url(${this.depotRessources.fondAccueil().src})`,
-      personnages: `url(${this.depotRessources.personnages().src})`,
-      forceCampagne: parsedUrl.searchParams.get('code') || ''
+      personnage: this.depotRessources.personnage().src,
+      precedent: this.depotRessources.precedent().src,
+      suivant: this.depotRessources.suivant().src,
+      punaise: this.depotRessources.punaise().src,
+      forceCampagne: parsedUrl.searchParams.get('code') || '',
+      indexBatiment: this.recupereNiveauDuPrecedentChargement()
     };
   },
 
-
   computed: {
-    ...mapState(['situations', 'estConnecte'])
+    ...mapState(['situations', 'estConnecte']),
+    ...mapGetters(['niveauActuel']),
+
+    positionFond () {
+      return (this.indexBatiment - 1) * 30;
```

Comment:
Quand j'ai 4 situations, le scrolling du fond est fluide, mais si j'en ai 3, il est saccadé.

Reply:
Comme on l'a vu ensemble, ça arrive sur mon ordinateur, un mac, avec firefox. C'est aussi un peu visible avec Chrome et Safari mais beaucoup moins.

J'ai fais un petit test en remplaçant la position de % en px et ça m'a l'aire un peu moins saccadé.
`return -(this.indexBatiment - 1) * LARGEUR_SCENE/3;`

Je suis supris d'être obligé de passer en négatif pour que le fond bouge dans le bon sens.

J'ai remarqué que changer de 3 à 4 situations augmente la taille du fond qui se retrouve à dépasser la taille de l'image de fond (avec 3 situations il n'y a pas de répétition de l'image de fond alors qu'avec 4 si). Je ne sais pas si c'est la raison des sacades et si c'est une bonne idée d'essayer d'exploité ça.

Reply:
bon, après plus de test, % ou px ça m'a l'air pareil.